docs/pywbemcligeneraloptions.rst

```diff
@@ -61,166 +70,470 @@ command, general the options redefinition's are not retained between command
 executions.
 
 
+.. _`Defining the WBEM Server`:
+
+Defining the WBEM Server
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The target WBEM server  can be defined on the command line in several ways with
+the in the following arguments :
+
+1. Define the WBEM server characteristics with a set of general options
+   The geneal options that define the WBEM Server include the:
+
+   * The :ref:`--server general option` that defines the URI of the WBEM server
+   * The :ref:`--default-namespace general option`, that defines the nameespace
+     used if no namespace is included with each command,
+   * several connection security general options:
+
+     * The :ref:`--user general option`,  defines the WBEM server user name
+     * The :ref:`--password general option`, defining the password for the WBEM
+       server user
+     * The :ref:`--no-verify general option` that defines whether the client verifies
+       certificates received from the WBEM server,
+     * The :ref:`--certfile general option`, client certificate file,
+     * The :ref:`--keyfile general option`,  client key file,
+     * The :ref:`--ca-certs general option`), a collection of certificates against
+       which certificates received from the WBEM server are verified.
+
+   * The :ref:`--timeout general option` that defines the client side connection
+     timeout
+
+2. Define a mock-server. A mock server see :ref:`Mock WBEM server support`
+   substitutes a local mock WBEMConnection for a WBEM server and allows
+   testing or demonstrating pywbemcli without having access to a real WBEM
+   server.
+
+3. Define the characteristics for either a WBEM server or mock WBEM server with
+   pywbemcli and add the data to a pywbemcli :term:`connections file`.
+   pywbemcli can access the server data by simply defining the name on with the
+   pywbem call.
+
+
```

Comment:
How does the information about pull operations in the following section relate to 4.1.1. "Using pywbem Pull Operations from pywbemcli"?
How does the information about logging in the following section relate to 3.6. "Pywbemcli defined logging"?

Reply:
These are duplication.  I propose merging 4.1 into 3.4.

Am going to review this in the morning because I would make a mistake tonight.

TODO

Reply:
I merged the pull stuff from 4.1 into 3.4.  However that leaves one piece of section 4.1 without any good location, the table of pywbemcli commands vs CIM-XML operations..  It should be in section 4 but not part of that section on special features. 

You say we have 3 sections, I merged 4.1 pull with 3.4 pool.  What is the third section on pull/traditional.

**DISCUSSION:** your comment 2, on logging. Not sure what you meant.

Reply:
On logging: There is also some duplication between here and the section I referenced.